library/coretests/tests/num/flt2dec/random.rs

```diff
@@ -105,6 +119,27 @@ where
     });
 }
 
+#[cfg(any(target_arch = "x86", all(target_arch = "aarch64", target_feature = "neon")))]
+pub fn f16_exhaustive_equivalence_test<F, G>(f: F, g: G, k: usize)
+where
+    F: for<'a> FnMut(&Decoded, &'a mut [MaybeUninit<u8>]) -> Option<(&'a [u8], i16)>,
+    G: for<'a> FnMut(&Decoded, &'a mut [MaybeUninit<u8>]) -> (&'a [u8], i16),
+{
+    // we have only 2^23 * (2^8 - 1) - 1 = 2,139,095,039 positive finite f16 values,
```

Comment:
How are there more values than bits in f16? Is this stale from f32 perhaps?

Reply:
Good catch, this was a copy+paste error. For f16 the test is actually almost instant, so it doesn't need to be ignored either except for miri.

Updated this.